pkg/apis/options/app.go

```diff
@@ -22,6 +22,12 @@ type Templates struct {
 	// password form if a static passwords file (htpasswd file) has been
 	// configured.
 	DisplayLoginForm bool `flag:"display-htpasswd-form" cfg:"display_htpasswd_form"`
+
+	// Debug renders detailed errors when an error page is shown.
+	// It is not advised to use this in production as errors may contain sensitive
+	// information.
+	// Use only for diagnosing backend errors.
+	Debug bool `flag:"show-debug-on-error" cfg:"show-debug-on-error"`
```

Comment:
I need to add this new flag to the docs... sigh, that list is too big!

Reply:
Hydra: Cut off one head two grow back in its place 😄 

It will only grow no matter how hard we try to slay it